spec/unit/actions/collect_data_product_spec.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+describe GoodData::LCM2::CollectDataProduct do
+  it 'Has GoodData::Bricks::DefaultDataProductMiddleware class' do
+    GoodData::LCM2::CollectDataProduct.should_not be(nil)
+  end
+
+  let(:client) { double(:client) }
+  let(:domain) { double(:domain) }
+  let(:data_product) { GoodData::DataProduct.new({}) }
+  let(:gdc_logger) { double(:gdc_logger) }
+
+  before do
```

Comment:
Identical code found in collect_data_product_spec.rb:17, ensure_data_product_spec.rb:16 (mass = 64)